src/topology/instances/real_vector_space.lean

```diff
@@ -5,6 +5,7 @@ Authors: Yury Kudryashov
 -/
 import topology.algebra.module.basic
 import topology.instances.real
```

Comment:
```suggestion
```

Reply:
I'd argue that this is better as is, as it encodes "we use lemmas both from the real and rational instances". This makes it easier to shuffle around the imports again in future.

Reply:
ok fair. I thought we usually got rid of transitive dependencies, but I don't mind keeping it.